### DISCUSSION TITLE
Finality

### DISCUSSION THREAD

**Discussion by mzeiya** [Cyfrin/foundry-full-course-cu#2075] [General]
cant the finality aspect where a transaction has been confirmed but some processing going on, be a problem or an avenue for fraud?

**Comment by EngrPips** [comment]
If a transaction that isn't finalized turns out to be a false transaction, then that transaction gets reverted, and every other transaction that that transaction is linked to also gets reverted. You should read this Ethereum stack exchange post for more details `https://ethereum.stackexchange.com/a/164704/91017`
